pkg/reporter/json.go

```diff
@@ -37,6 +37,7 @@ type file struct {
 	SpecVersion string   `json:"spec_version"`
 	Format      string   `json:"file_format"`
 	AvgScore    float64  `json:"avg_score"`
+	Components  int      `json:"no_components"`
```

Comment:
Name - `no_components` can be confused with components' availability. How about `num_components`?